src/shared/utils/exportData/getDrawingLines.ts

```diff
@@ -1,8 +1,25 @@
-import { DecryptedDrawingValue } from 'shared/types';
+import { DecryptedDrawingValue, ExportAnswer } from 'shared/types';
+
+// Ex. appVersion = '1.0.4' ... '1.0.7'; appId: "mindlogger-mobile"
+export const checkIf104_107MobileAppVersion = (client: ExportAnswer['client']) => {
```

Comment:
Maybe it make sense to think of changing variable name? for example, checkIfShouldScaleCoords or smth like that? 104_107 looks strange (and the same for has104_107... flag)